### PR TITLE
Revert change to remove encoding of handles out of the DDS

### PR DESCRIPTION
**Pull request by jzaffiro**
Reverts #19242 due to a bug where duplicate attach ops are sent because of the change to encoding handles.

### CODE REVIEW

File: packages/dds/matrix/api-report/matrix.api.md [hunk 1]
```diff
@@ -6,26 +6,24 @@
...
 import { IChannelFactory } from '@fluidframework/datastore-definitions';
 import { IChannelServices } from '@fluidframework/datastore-definitions';
-import { IChannelStorageService } from '@fluidframework/datastore-definitions/internal';
-import { IEvent } from '@fluidframework/core-interfaces/internal';
-import { IEventProvider } from '@fluidframework/core-interfaces/internal';
-import { IEventThisPlaceHolder } from '@fluidframework/core-interfaces/internal';
+import { IChannelStorageService } from '@fluidframework/datastore-definitions';
+import { IEvent } from '@fluidframework/core-interfaces';
+import { IEventProvider } from '@fluidframework/core-interfaces';
+import { IEventThisPlaceHolder } from '@fluidframework/core-interfaces';
```
Comment (markfields) [inline packages/dds/matrix/api-report/matrix.api.md]: So did these `/internals` show up somewhere along the RC2 release branch?  Or after his commit or something?  Surprised to see it, but if the checks pass it'll be fine

File: packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import { pkgVersion } from "./packageVersion.js";
 
 /**
  * The factory that defines the consensus queue
+ * @internal
```
Comment (markfields) [inline packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts:17]: Why the mismatch between the class and the functions below?  Seems off.  I thought classes/members had to all have the same tags.

Comment (jzaffiro) [reply packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts:17]: Just an oversight, I'll update it

File: packages/dds/matrix/api-report/matrix.api.md (hunk 1 above)
Comment (jzaffiro) [reply packages/dds/matrix/api-report/matrix.api.md]: I would assume they came at a later point in the branch? I kept getting errors on any that I left in, but I was surprised too.

**Review by markfields — APPROVED** [review]

File: packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts (hunk 2 above)
Comment (jzaffiro) [reply packages/dds/ordered-collection/src/consensusOrderedCollectionFactory.ts:17]: Updated to match what was in main/rc2 branches before: class tagged `@internal`, no tags on the functions

File: packages/dds/register-collection/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ library consumers should always prefer `^`.
 
 ### Detecting concurrency in Fluid
 
-In distributed systems literaure, detecting concurrency requires some form of logical/phsical clock. A popular technique used in replicated databases such as dynamodb is called version vectors where each key stores a collection of `[time, value]` tuples. `time` is essentially a reference clock used to decide concurrency amongst updates. Each update to a key includes the `time`, essentially to indicate how `caught up` the replica was during that update.
+In distributed systems literaure, detecting conucurrency requires some form of logical/phsical clock. A popular technique used in replicated databases such as dynamodb is called version vectors where each key stores a collection of `[time, value]` tuples. `time` is essentially a reference clock used to decide concurrency amongst updates. Each update to a key includes the `time`, essentially to indicate how `caught up` the replica was during that update.
```
Comment (markfields) [inline packages/dds/register-collection/README.md:22]: Oops? :)
```suggestion
In distributed systems literaure, detecting concurrency requires some form of logical/phsical clock. A popular technique used in replicated databases such as dynamodb is called version vectors where each key stores a collection of `[time, value]` tuples. `time` is essentially a reference clock used to decide concurrency amongst updates. Each update to a key includes the `time`, essentially to indicate how `caught up` the replica was during that update.
```

Comment (markfields) [reply packages/dds/register-collection/README.md:22]: Oh never mind, probably Connor fixed the typo and you're reverting it.  Leave it.

Comment (markfields) [reply packages/dds/register-collection/README.md:22]: I thought it was a case where your cursor was in a different window than you thought, that happens to be pretty regularly.

**Review by anthony-murphy — APPROVED** [review]

**Pull request merged**